heft-plugins/heft-jest-plugin/src/JestPlugin.ts

```diff
@@ -379,6 +399,101 @@ export class JestPlugin implements IHeftPlugin<IJestPluginOptions> {
     ];
   }
 
+  // Resolve all specified properties using Node resolution, and replace <rootDir> with the same rootDir
+  // that we provide to Jest. Resolve if we modified since paths containing <rootDir> should be absolute.
+  private static _getJsonPathMetadata(options: IJsonPathMetadataOptions): IJsonPathMetadata {
+    return {
+      customResolver: (configurationFilePath: string, value: string) => {
+        const configurationFileDir: string = path.dirname(configurationFilePath);
+        let packageDirMatches: RegExpExecArray | null;
+
+        // Compare with replaceRootDirInPath() from here:
+        // https://github.com/facebook/jest/blob/5f4dd187d89070d07617444186684c20d9213031/packages/jest-config/src/utils.ts#L58
+        if (value.startsWith(JestPlugin._rootDirToken)) {
+          // Example:  <rootDir>/path/to/file.js
+          const restOfPath: string = path.normalize('./' + value.substr(JestPlugin._rootDirToken.length));
```

Comment:
The `normalize` calls may be unnecessary.

Reply:
I'm going to leave it this way, only because that's what happens inside of Jest and I don't want to tweak that. This came in after @octogonz had an issue with how rootDir was substituted in, so I'd err on the side of "safe"